test/put.sh

```diff
@@ -274,6 +274,43 @@ it_can_put_to_url_with_merge_commit() {
   test $latest_merge_ref = $merged_ref
 }
 
+it_chooses_the_unmerged_commit_ref() {
```

Comment:
You'll need a `run it_chooses_the_unmerged_commit_ref` line at the bottom of the file in order to run the test. :slightly_smiling_face: